src/GafferScene/Preview/RendererAlgo.cpp

```diff
@@ -206,8 +204,7 @@ struct RenderSets::Updater
 
 	void operator()( const tbb::blocked_range<size_t> &r )
 	{
-		ContextPtr context = new Context( *m_context, Context::Borrowed );
-		Context::Scope scopedContext( context.get() );
+		Context::EditableScope context( m_context );
```

Comment:
Same thought about using `SetScope` to clarify intentions. Actually, are we missing the removes here?